hack/test-e2e-local.sh

```diff
@@ -19,6 +19,12 @@ ginkgo_flags=
 if [ -n "${CI:-}" -a -n "${ARTIFACTS:-}" ] ; then
   mkdir -p "$ARTIFACTS"
   ginkgo_flags="--output-dir=$ARTIFACTS --junit-report=junit.xml"
+  
+  printf "\n127.0.0.1 api.e2e-default.local.external.local.gardener.cloud\n127.0.0.1 api.e2e-default.local.internal.local.gardener.cloud\n" >> /etc/hosts
+else
+  if ! grep -q "127.0.0.1 api.e2e-default.local.external.local.gardener.cloud" /etc/hosts ; then
+    printf "To access the shoot cluster and running e2e tests, you have to extend your /etc/hosts file.\nPlease refer https://github.com/gardener/gardener/blob/master/docs/deployment/getting_started_locally.md#accessing-the-shoot-cluster"
```

Comment:
```suggestion
    printf "To access the shoot cluster and running e2e tests, you have to extend your /etc/hosts file.\nPlease refer https://github.com/gardener/gardener/blob/master/docs/deployment/getting_started_locally.md#accessing-the-shoot-cluster\n"
```

as suggested in https://github.com/gardener/gardener/pull/5938#discussion_r870046546

Reply:
I think he suggested to add the new line in `/etc/hosts` file.

Reply:
Right 😄